frontend/src/ui/metadata.tsx

```diff
@@ -42,18 +42,17 @@ export const SmallDescription: React.FC<SmallDescriptionProps> = ({
     const sharedStyle = {
         fontSize: 13,
         marginTop: 4,
+        color: COLORS.grey6,
     };
 
     if (text === null) {
         return <div {...{ className }} css={{
             ...sharedStyle,
-            color: COLORS.grey5,
```

Comment:
I'm on the fence about this. The contrast between `grey5` and the background is too low in both light and dark mode, hence this change. 
I do prefer the "no description" being less prominent than an actual description. But I also don't really like making the description itself any brighter. Sooo as always, open for suggestions.

Reply:
Mh wow, yeah, the contrast to the background really changes dramatically between grey5 and grey6. Interesting. And unfortunate. I agree in all points with you: I don't see a particularly good solution. Apart from introducing another grey tone. I'd say: leave it as it is now (in this PR). Then at least we have sufficient contrast. We can still think of ways to distinguish it more from the actual description.